src/components/ConnectToQuickbooksOnlineButton/index.native.tsx

```diff
@@ -22,38 +21,36 @@ type ConnectToQuickbooksOnlineButtonOnyxProps = {
     session: OnyxEntry<Session>;
 };
 
+const renderLoading = () => <FullScreenLoadingIndicator />;
+
 function ConnectToQuickbooksOnlineButton({
     policyID,
     session,
-    disconnectIntegrationBeforeConnecting,
+    shouldDisconnectIntegrationBeforeConnecting,
     integrationToDisconnect,
 }: ConnectToQuickbooksOnlineButtonProps & ConnectToQuickbooksOnlineButtonOnyxProps) {
-    const styles = useThemeStyles();
     const {translate} = useLocalize();
     const webViewRef = useRef<WebView>(null);
     const [isWebViewOpen, setWebViewOpen] = useState(false);
 
     const authToken = session?.authToken ?? null;
 
-    const renderLoading = () => <FullScreenLoadingIndicator />;
-
     const [isDisconnectModalOpen, setIsDisconnectModalOpen] = useState(false);
 
     return (
         <>
             <Button
                 onPress={() => {
-                    if (disconnectIntegrationBeforeConnecting && integrationToDisconnect) {
+                    if (shouldDisconnectIntegrationBeforeConnecting && Boolean(integrationToDisconnect)) {
                         setIsDisconnectModalOpen(true);
                         return;
                     }
                     setWebViewOpen(true);
                 }}
                 text={translate('workspace.accounting.setup')}
-                style={styles.justifyContentCenter}
```

Comment:
We deleted this style caused a regression [here](https://github.com/Expensify/App/issues/41301)